resources.html

```diff
@@ -42,223 +19,40 @@
 
 <h2>Other Resources</h2>
 
-<p>Omnigraffle&reg; and Word&reg; files used for the creation of the screenshots used in the User Help website and for the PDF versions for printing.</p>
-
-<p>These source files can be downloaded from the Resources directory in OME Downloads:</p>
+<p>Omnigraffle&reg; files used for the creation of the screenshots used in the User Help website and other resources such as Word&reg; files can be downloaded from the Help resources directory on the OME Downloads site:</p>
```

Comment:
Fixed now, sorry this slipped off my radar yesterday @jburel - are you happy with all the rest of the content on this page that I've deleted going? Can I delete the images now too?